curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/add-images-to-your-website.md

```diff
@@ -25,7 +27,9 @@ Ideally the `alt` attribute should not contain special characters unless needed.
 
 Let's add an `alt` attribute to our `img` example above:
 
-`<img src="https://www.freecatphotoapp.com/your-image.jpg" alt="A business cat wearing a necktie.">`
+```html
+<img src="https://www.freecatphotoapp.com/your-image.jpg" alt="A business cat wearing a necktie.">
+```
 
 # --instructions--
 
```

Comment:
One more on line 42 of this file (GitHub won't let me comment that far away from the diff).

Reply:
Good catch

Reply:
@Sembauke Just to clarify what @nhcarrigan and I are suggesting:  Line 40 would become:
```md
Now set the `src` attribute so that it points to the url `https://bit.ly/fcc-relaxing-cat`
```
and the existing line 41 and 42 would be removed.